### PR TITLE
Added RVD_HTTP_TIMEOUT configuration option to advanced.conf and conf…

### PR DESCRIPTION
**Pull request by otsakir**
…igured rvd.xml

Refers #2718

Relates to #2711

### CODE REVIEW

File: restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-rvd.sh [hunk 1]
```diff
@@ -41,9 +41,28 @@ updateMaxMediaFileSize() {
...
+          mv ${RVD_XML_FILE}_tmp ${RVD_XML_FILE}
+      fi
+      echo "set defaultHttpTimeout to $1"
+  fi
+}
+
 # MAIN
 
-if [[ "$RVD_UNDEPLOY" = true || "$RVD_UNDEPLOY" = TRUE || "$RVD_UNDEPLOY" != True ]]; then
+if [[ "$RVD_UNDEPLOY" = true || "$RVD_UNDEPLOY" = TRUE || "$RVD_UNDEPLOY" = True ]]; then
```
Comment (jaimecasero) [inline restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-rvd.sh]: good catch here

File: restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf [hunk 2]
```diff
@@ -97,6 +97,7 @@ RVD_UNDEPLOY='false'
 RVD_URL='' # override if RVD is not under the same domain with restcomm. Use a base url like http://myrvd.restcomm.com . No path included. This affects restcomm.xml/<rcmlserver/>
 RVD_VIDEO_SUPPORT='false'  # Toggle RVD Video RCML generation and UI
 RVD_MAX_MEDIA_FILE_SIZE=4194304 # Limit media file size when uploading. For video should probably be greater
+RVD_HTTP_TIMEOUT=''  # Timeout for http connections in rvd http client. It mainly affects connections to restcomm (create calls, fetch account etc.). It does not affect ES.
```
Comment (jaimecasero) [inline restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf]: do we need default value?. empty is using hardcoded? can you add comment with expected time units(secs,millis....)?

Comment (otsakir) [reply restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf]: @jaimecasero , is empty the setting (if exists) will be removed from rvd.xml and the default value in the rvd binary will be used.

Comment (otsakir) [reply restcomm/configuration/config-scripts/as7-config-scripts/restcomm/advanced.conf]: Let  me add the time units (ms)

**Comment by jaimecasero** [comment]
im ok merging this

File: restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-rvd.sh (hunk 1 above)
Comment (otsakir) [reply restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-rvd.sh]: ;-) ran some manual testing and fell right on it.

**Pull request merged**